src/styles/blocks/general.scss

```diff
@@ -0,0 +1,31 @@
+html {
+  font-family: Roboto, Arial, sans-serif;
+  font-size: 12px;
+  font-weight: 400;
+  font-style: normal;
+}
+
+body {
+  margin: 0;
+}
+
+p,
+h2 {
+  margin: 0;
+}
+
+a.is-active::after {
+  content: '';
+  position: absolute;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  height: 4px;
+  border-radius: 2px;
+  background-color: $main-color;
+}
+
+a.is-active,
+a.is-active:hover {
+  color: $main-color;
```

Comment:
Ensure that the hover color for navigation links is set to `#00acdc` as specified in the task requirements. Also, make sure the base color for nav links is `#060b35`.